src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -1946,24 +1946,23 @@ static void Main()
         public void InvalidTypeArguments()
         {
             var source =
-@"unsafe class Program
+@"using System;
+unsafe class Program
 {
     static int* F() => throw null;
     static void Main()
     {
-        System.Delegate d;
-        d = F;
-        d = (int x, int* y) => { };
+        Delegate d1 = F;
+        Delegate d2 = (int x, int* y) => { };
+        Report(d1);
+        Report(d2);
     }
+    static void Report(Delegate d) => Console.WriteLine(d.GetType());
 }";
-            var comp = CreateCompilation(source, parseOptions: TestOptions.RegularPreview, options: TestOptions.UnsafeReleaseExe);
-            comp.VerifyDiagnostics(
-                // (7,13): error CS8917: The delegate type could not be inferred.
-                //         d = F;
-                Diagnostic(ErrorCode.ERR_CannotInferDelegateType, "F").WithLocation(7, 13),
-                // (8,13): error CS8917: The delegate type could not be inferred.
-                //         d = (int x, int* y) => { };
-                Diagnostic(ErrorCode.ERR_CannotInferDelegateType, "(int x, int* y) => { }").WithLocation(8, 13));
+            CompileAndVerify(source, parseOptions: TestOptions.RegularPreview, options: TestOptions.UnsafeReleaseExe, expectedOutput:
```

Comment:
>expectedOutput [](http://example.com/codeflow?start=118&length=14)

We probably should invoke the delegates and observe success.

Reply:
> We probably should invoke the delegates and observe success.

Even with the change, it is not observable if the right code got executed. Perhaps the target function/lambda can print something?